AWSCreateStack/Program.cs

```diff
@@ -33,6 +34,9 @@ class Options
         [Option('t', "templatePath", Required = true, HelpText = "Path to cloud formation template")]
         public string TemplatePath { get; set; }
 
+        [Option('p', "parameterPath", Required = false, HelpText = "Path to template parameters")]
+        public string ParameterPath { get; set; }

```

Comment:
Should this be plural "parametersPath"? Because it's not a single parameter in this file.